tests/foreman/api/test_errata.py

```diff
@@ -183,9 +188,7 @@ def test_positive_install_in_hc(module_org, activation_key, custom_repo, target_
 
     :BZ: 1983043
     """
-    with VMBroker(
-        nick=constants.DISTRO_RHEL7, host_classes={'host': ContentHost}, _count=2
-    ) as clients:
+    with VMBroker(nick=distro, host_classes={'host': ContentHost}, _count=2) as clients:
```

Comment:
Here, instead of calling VMBroker directly, you could try using the [content_hosts](https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/core/contenthosts.py#L93) fixture which also returns 2 hosts and use it with a loop similarly `for client in content_hosts:`